__tests__/unit/formatter-test.ts

```diff
@@ -0,0 +1,95 @@
+         25:21  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         26:19  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         32:34  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+
+      /Users/fake/app/initializers/tracer.js
+         1:11  error  'window' is already defined as a built-in global variable          no-redeclare
+         1:19  error  'XMLHttpRequest' is already defined as a built-in global variable  no-redeclare
+
+      /Users/fake/app/models/build.js
+         108:50  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         120:25  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+
+      /Users/fake/app/services/insights.js
+         287:17  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         296:17  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         307:17  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         317:17  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+
+      /Users/fake/app/utils/traverse-payload.js
+         18:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+
+      /Users/fake/tests/unit/services/insights-test.js
+          65:27  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+          80:27  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+          97:27  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         116:27  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         134:27  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+         158:32  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
+
+      ✖ 18 problems (18 errors, 0 warnings)
+        1 error and warnings potentially fixable with the \`--fix\` option.
+
+      "
+    `);
+  });
+
+  it('should not return anything when shouldIncludeTodo is false and there are only todo items', async () => {
+    const results = fixtures.eslintWithTodos();
+
+    expect(stripAnsi(formatter(results)).trim()).toEqual('');
+  });
+
+  it('should return all todo items when shouldIncludeTodo is true', async () => {
+    const results = fixtures.eslintWithTodos();
+
+    expect(stripAnsi(formatter(results, { shouldIncludeTodo: true })))
```

Comment:
See my comment in `formatter.ts` below.

```suggestion
    expect(stripAnsi(formatter(results, { includeTodo: true })))
```